src/garth/sso.py

```
@@ -33,7 +33,13 @@ def __init__(
     ):
         global OAUTH_CONSUMER
         if not OAUTH_CONSUMER:
-            OAUTH_CONSUMER = requests.get(OAUTH_CONSUMER_URL).json()
+            request_kwargs: dict[str, Any] = {}
+            if parent is not None:
+                request_kwargs["proxies"] = parent.proxies
+                request_kwargs["verify"] = parent.verify
+            OAUTH_CONSUMER = requests.get(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)